app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Task;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator;
+
+class TaskController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $tasks = Task::orderBy('created_at', 'asc')->paginate(5);
+        return view('tasks', compact('tasks'));
```

Comment:
```suggestion
        $tasks = Task::orderBy('created_at', 'asc')->paginate(config('app.default_paginate'));

        return view('tasks', compact('tasks'));
```
bn để số paginate vào config nhé.